packages/dev-tools/pages/index.js

```diff
@@ -459,27 +460,37 @@ export default class IndexPage extends React.Component {
     super(props);
     this.state = {
       disconnected: false,
+      client: null,
     };
     this.unsubscribers = [];
     if (process.browser) {
-      this.subscriptionClient = new SubscriptionClient(`ws://${window.location.host}/graphql`, {
-        reconnect: true,
-      });
-      this.client = createApolloClient(this.subscriptionClient);
+      this.connect().catch(error => this.setState({ disconnected: true }));
```

Comment:
I think we should be doing this work in `componentDidMount`, not the constructor.

Reply:
You're right, fixed now.